shell/platform/android/io/flutter/embedding/engine/systemchannels/TextInputChannel.java

```diff
@@ -328,6 +345,16 @@ public void setTextInputMethodHandler(@Nullable TextInputMethodHandler textInput
 
     // TODO(mattcarroll): javadoc
     void clearClient();
+
+    /**
+     * Sends app private command to the current Input Method.
+     *
```

Comment:
Add a link to some documentation please since it's a bit more obscure

Reply:
Done